deploy/stacks/pipeline.py

```diff
@@ -558,6 +559,7 @@ def set_quality_gate_stage(self):
                         'mkdir -p source_build',
                         'mv backend ./source_build/',
                         'mv config.json ./source_build/',
+                        'mv version.json ./source_build/',
```

Comment:
Do we need this line for `UploadCodeToS3`? I think we can remove as the `source_build.zip` is used primarily for DBMigration step only which should not be dependent on version

Reply:
Let's remove this line `mv version.json ./source_build/` - I tested to ensure it is not needed

I think best to keep hardcoding of `version.json` minimal to only when necessary

Will approve right after this change

Reply:
Done